x-pack/legacy/plugins/siem/public/components/formatted_date/index.test.tsx

```diff
@@ -6,174 +6,166 @@
 
 import { mount, shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
-import moment from 'moment-timezone';
 import * as React from 'react';
 
-import { useUiSetting$ } from '../../lib/kibana';
+import { useDateFormat, useTimeZone } from '../../hooks';
 
-import { mockFrameworks, TestProviders, MockFrameworks, getMockKibanaUiSetting } from '../../mock';
+import { TestProviders } from '../../mock';
 import { getEmptyString, getEmptyValue } from '../empty_value';
 import { PreferenceFormattedDate, FormattedDate, FormattedRelativePreferenceDate } from '.';
 
-jest.mock('../../lib/kibana');
-const mockUseUiSetting$ = useUiSetting$ as jest.Mock;
+jest.mock('../../hooks');
+const mockUseDateFormat = useDateFormat as jest.Mock;
+const mockUseTimeZone = useTimeZone as jest.Mock;
+
+const isoDateString = '2019-02-25T22:27:05.000Z';
 
 describe('formatted_date', () => {
+  let isoDate: Date;
+
+  beforeEach(() => {
```

Comment:
do we really need to do that in the beforeEach since it seems just be static

Reply:
Probably not, although it does prevent mutations to `isoDate` from affecting other tests. I've been bit by that a few times, so I typically do test setup like this. Happy to change it if you feel strongly!